README.md

```diff
@@ -1,45 +1,17 @@
 # Using u-boot on Creator (Ci40) Marduk platform
 
-### How to build/cross-compile for Ci40:
+## Setup your toolchain
+If using the toolchain produced by openwrt you must setup the enviroment as expained
```

Comment:
Spell check and grammar:
1. enviroment => environment
2. expained => explained
3. setup => set up
4. If using... => If you are using...
5. Setup your toolchain => Toolchain setup OR Set up your toolchain
6. openwrt => OpenWrt